src/OrbitBase/SimpleExecutor.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitBase/SimpleExecutor.h"
+
+#include <memory>
+
+namespace orbit_base {
+void SimpleExecutor::ScheduleImpl(std::unique_ptr<Action> action) {
+  scheduled_tasks_.emplace_back(std::move(action));
```

Comment:
MutexLock

Reply:
Ahhh yes. Thanks for spotting this. Fix: #2411